app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -258,10 +262,16 @@ public void removeTextInputListener(GeckoSession.TextInputDelegate aListener) {
         mTextInputListeners.remove(aListener);
     }
 
-    public static class SessionSettings {
+    public class SessionSettings {
         public boolean multiprocess = false;
         public boolean privateMode = false;
         public boolean trackingProtection = true;
+        public int userAgentMode;
+
+        public SessionSettings() {
+            boolean isDesktopMode = SettingsStore.getInstance(mContext).isDesktopVersionEnabled();
```

Comment:
@thenadj This will require new design as in the original spec there were only Desktop/Vr options and was designed as a switch. Ideas: radio button or dropdown menu.